src/NuGet.Clients/NuGet.VisualStudio.Common/VsHierarchyUtility.cs

```diff
@@ -19,6 +19,11 @@ public static class VsHierarchyUtility
     {
         private const string VsWindowKindSolutionExplorer = "3AE79031-E1BC-11D0-8F78-00A0C9110057";
 
+        private static readonly HashSet<string> UnsupportedProjectCapabilities = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
I don't see this being used as a hash set or to compare, I only see it used for iterating, i would make this an array instead